pkgs/top-level/python-packages.nix

```
@@ -5153,6 +5153,13 @@ in {
 
   transitions = callPackage ../development/python-modules/transitions { };
 
+  essentia = (toPythonModule (pkgs.essentia.override {
+    pythonPackages = self;
```

instead of passing the package scope, you should pass the interpreter, then pythonPackages -> python.pkgs
```suggestion
    inherit python;
```

---

Inheriting `python` works however. Marking this as resolved, yet not committing & pushing.